src/main/java/com/voorbeeld/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,51 @@
+package com.voorbeeld.TechItEasy.controllers;
+
+import com.voorbeeld.TechItEasy.exceptions.RecordNotFoundException;
+import com.voorbeeld.TechItEasy.model.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+
+@RestController
```

Comment:
je kunt hier @RequestMapping("/televisions") toevoegen dan kun je "/televisions" dan starten alle endpoints in  deze controller standaard met "/televisions"